### PR TITLE
[chassis][202405]: Remove default hash-offset for chassis-packet

### PR DESCRIPTION
**Pull request by anamehra**
Signed-off-by: anamehra@cisco.com

#### Why I did it
Fixes https://github.com/sonic-net/sonic-buildimage/issues/19746

switch.json in swss docker sets the default hash offset to 0 for chassis-packet. The change was recently introduced via https://github.com/sonic-net/sonic-buildimage/pull/18912 which was supposed to be only for T0/T1. On chassis-packet, each asics needs to set an unique hash offset to set an unique offset for load balancing algorithm. The new change overwrites platform setting and resets the hash-offset to 0 on all asics. This breaks the ecmp load balancing algorithm on chassis-packet.

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Remove the default hash-offset setting for chassis-packet. This setting is provided by platform based on platform specific rules.

#### How to verify it
Run sonic-mgmt fib/test_fib.py

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305
- [x] 202405

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

File: dockers/docker-orchagent/switch.json.j2 [hunk 1]
```diff
@@ -26,8 +26,10 @@
             "ecmp_hash_seed": "{{ hash_seed_value }}",
             "lag_hash_seed": "{{ hash_seed_value }}",
             "fdb_aging_time": "600",
+{% if not DEVICE_METADATA.localhost.switch_type or DEVICE_METADATA.localhost.switch_type != "chassis-packet" %}
```
Comment (abdosi) [inline dockers/docker-orchagent/switch.json.j2:29]: @anamehra : change this condition to be more platform specific where if asic_type is `cisco-8000` and topology is  `t2` we don;t set hash offset explicitly

**Review by abdosi — APPROVED** [review]

**Review by yejianquan — APPROVED** [review]
LGTM

**Review by wangxin — APPROVED** [review]

**Pull request merged**